src/Umbraco.Infrastructure/Persistence/Repositories/Implement/MemberRepository.cs

```diff
@@ -314,6 +314,7 @@ protected override void PersistNewItem(IMember entity)
             // persist the member dto
             dto.NodeId = nodeDto.NodeId;
 
+            // TODO: password parts of this file need updating
```

Comment:
I found a bit of legacy password functionality in `PersistNewItem` - does it need changing now?

Reply:
This password logic here can stay. It's purely used for dealing with flagging a member as not having a set password so the value uses a special prefix. The usage of the IPasswordHasher here is irrelevant, it might as well be a SHA hash but it's fine the way it is.